pkg/generate/code/set_sdk.go

```diff
@@ -784,8 +784,8 @@ func setSDKReadMany(
 			}
 		}
 
-		// Field renames are handled in getSanitizedMemberPath
-		resVarPath, err = getSanitizedMemberPath(memberName, r, op, sourceVarName)
+		// Field renames are handled in GetSanitizedMemberPath
+		resVarPath, err = r.GetSanitizedMemberPath(memberName, op, sourceVarName)
 		if err != nil {
```

Comment:
Just double checking: Why do we not just do `if err == nil { continue }` here, instead?